src/components/navigation/course-footer.tsx

```diff
@@ -5,8 +5,10 @@ import styled from 'styled-components'
 
 import { makeMargin, makeDefaultButton } from '../../helper/css'
 
-// TODO: needs type declaration
-export type CourseFooterProps = any
+export interface CourseFooterProps {
+  opener: (e: React.MouseEvent<HTMLAnchorElement>) => void
```

Comment:
would probably rename that to `on*` to be consistent with other event handlers. Haven't looked too much into the code to know when this is actually triggered